src/lando/headless_api/tests/test_automation_api.py

```diff
@@ -589,6 +589,109 @@ def test_automation_job_add_commit_fail(
     assert scm.push.call_count == 0
 
 
+PATCH_DIFF = """
+diff --git a/test.txt b/test.txt
+--- a/test.txt
++++ b/test.txt
+@@ -1,1 +1,2 @@
+ TEST
++adding another line
+""".lstrip()
+
+
+@pytest.mark.django_db
+def test_automation_job_create_commit_success_hg(
+    hg_server, hg_clone, repo_mc, treestatusdouble, hg_automation_worker, monkeypatch
+):
+    repo = repo_mc(SCM_TYPE_HG)
+    scm = repo.scm
+
+    # Create a job and actions
+    job = AutomationJob.objects.create(
+        status=JobStatus.SUBMITTED,
+        requester_email="example@example.com",
+        target_repo=repo,
+    )
+    AutomationAction.objects.create(
+        job_id=job,
+        action_type="create-commit",
+        data={
+            "action": "create-commit",
+            "author": "Test User <test@example.com>",
+            "commit_message": "add another file",
+            "date": 0,
+            "diff": PATCH_DIFF,
+        },
+        order=0,
+    )
+
+    hg_automation_worker.worker_instance.applicable_repos.add(repo)
+
+    # Mock `phab_trigger_repo_update` so we can make sure that it was called.
+    mock_trigger_update = mock.MagicMock()
+    monkeypatch.setattr(
+        "lando.api.legacy.workers.automation_worker.AutomationWorker.phab_trigger_repo_update",
+        mock_trigger_update,
+    )
```

Comment:
out of scope, but noticing this repeated in many places, might make sense to just add it to the fixture, and maybe pass a flag to disable it if needed.

Reply:
File https://bugzilla.mozilla.org/show_bug.cgi?id=1960790 for this. :)